desktop/linux/index.md

```diff
@@ -144,7 +160,12 @@ $ sudo apt remove docker-desktop
 
 ## Known issues
 
- - The **Reset to factory defaults** option on the **Troubleshoot** page currently does not work.
+ - Docker CLI login flow presents some inconsistencies that are under investigation. If problems are encountered when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either 
+ `systemctl --user restart docker-desktop` or quit Docker Desktop and relaunch).
+   * Docker Desktop stores the passwords in base-64 encoded plaintext, integration with `pass` is work in progress.
```

Comment:
What happens after the release, will we store all the password in pass, and remove that in the credStore?

Reply:
WDYM @fredericdalleau? `pass` is the credStore

Reply:
Yes, this is the current plan.

Reply:
Plaintext is saved as base64 "user:pass" in config.json when we don't use any credStore

Reply:
Yes I mean remove the paintext password. We don't want a user that installs the tech preview, and later updates his docker and still have a password stored in plaintext (or base64 it's the same) left on the host.
HOwever, if it wasn't clear, I'm strongly against storing the password in plain text or base64. Just do the pass integration and we are done. The risk isn't worth the reward imho.